backend/src/user_commands.py

```diff
@@ -304,20 +296,13 @@ def CMD_CommitSell(cmdDict, threadContext):
 
     # The users account is modified after committing to sell the stock, log this action
     cmdDict['timestamp'] = str(int(time.time()*1000))
-
-    if not threadContext["sellAmtQ"].empty():
-        threadContext["sellAmtQ"].put(cmdDict['amount'])
-        log.logEvents['userCommand'](cmdDict)
-    else: 
-        cmdDict['amount'] = "0.00"
-        cmdDict['errorMessage'] = "Invalid cmd. No recent pending buys" 
-
+    log.logEvents['userCommand'](cmdDict)
```

Comment:
Dianna. It seems that we don't have a sellAmtQ, and buyAmtQ anymore.  Do we still need this line? 
`log.logEvents['userCommand'](cmdDict)`

Reply:
I think we still need to log those commands tho

Reply:
which commands? the else branch?

```
    else: 	
        cmdDict['amount'] = "0.00"	
        cmdDict['errorMessage'] = "Invalid cmd. No recent pending buys"
```

Reply:
I'm still figuring my way around mongo and sams changes, but I believe the triggers are handled in the db so yea the queues might not be needed , you're right

Reply:
I think the log is outside of the if else, so it logs it regardless. But the logging line can probably be moved to a better spot

Reply:
Sweet, thanks!  I'll keep them removed and hope nothing breaks :)

Reply:
:) fingers crossed!